lib/matplotlib/tests/test_image.py

```diff
@@ -789,9 +786,16 @@ def test_imshow_flatfield():
     im.set_clim(.5, 1.5)
 
 
-def test_empty_imshow():
+@pytest.mark.parametrize(
+    "make_norm",
+    [colors.Normalize,
+     colors.LogNorm,
+     lambda: colors.SymLogNorm(1),
+     lambda: colors.PowerNorm(1)])
+@pytest.mark.filterwarnings("ignore:Attempting to set identical left==right")
```

Comment:
For future reference, this was added in pytest 3.2 and our docs only say we require 3.0, so it probably shouldn't be relied upon.